workers/unity/Assets/Fps/Scripts/SetupLogic/WorkerConnectorBase.cs

```diff
@@ -9,6 +9,8 @@ public abstract class WorkerConnectorBase : DefaultWorkerConnector
     {
         public int TargetFrameRate = 60;
 
+        public MapBuilderSettings MapBuilderSettings;
```

Comment:
`[SerializeField] private`

Reply:
this actually needs to be protected, because the `SimulatedPlayerCoordinatorWorkerConnector` needs to be able to read the `MapBuilderSettings` to define its world bounds